core/tests/puppeteer-acceptance-tests/utilities/user/logged-out-user.ts

```diff
@@ -1109,10 +1102,11 @@ export class LoggedOutUser extends BaseUser {
       target => target.opener() === pageTarget
     );
     const newTabPage = await newTarget.page();
-    await newTabPage?.waitForNetworkIdle();
```

Comment:
Would it not be safer to let the waitForNetworkIdle() stay?

Reply:
Done